src/common/favaccordian.js

```diff
@@ -1,71 +1,71 @@
 import { useState } from "react";
 import * as moment from "moment";
-const Favaccordion = ({ children, props, sdata }) => {
+const Favaccordion = ({ children, props, secondflightdata }) => {
   const [isOpen, setOpen] = useState(false);
   let d1 = moment(props.data.departureTime);
   let d2 = moment(sdata.arrivalTime);
-  var startTime = moment(props.data.departureTime, "hh:mm:ss");
-  var endTime = moment(sdata.arrivalTime, "hh:mm:ss");
-  var diff = endTime.diff(startTime, "hours");
-  var totalMinutes = endTime.diff(startTime, "minutes");
-  var clearMinutes = totalMinutes % 60;
+  let startTime = moment(props.data.departureTime, "hh:mm:ss");
+  let endTime = moment(sdata.arrivalTime, "hh:mm:ss");
+  const diff = endTime.diff(startTime, "hours");
+  let totalMinutes = endTime.diff(startTime, "minutes");
+  let clearMinutes = totalMinutes % 60;
   const addtofavourite = () => {
-    var newArray = JSON.parse(sessionStorage.getItem("fav"));
-    //var newArray = [sessionStorage.getItem("fav")];
+    let newArray = JSON.parse(sessionStorage.getItem("fav"));
     newArray = newArray ? newArray : [];
-    //console.log(newArray);
     newArray = [...newArray, props];
     sessionStorage.setItem("fav", JSON.stringify(newArray));
-    console.log(JSON.parse(sessionStorage.getItem("fav")));
   };
   const clear = () => {
     sessionStorage.clear();
   };
-  if (diff > 0) {
-    return (
-      <div className="accordion-wrapper">
-        <div className={`accordion-title ${isOpen ? "open" : ""}`}>
-          <div className="row flex-container">
-            <div className="borderbox">
-              <div>
-                <span>Multiple</span>
+  return (
+    <div>
+      {diff ? (
+        <div className="accordion-wrapper">
+          <div className={`accordion-title ${isOpen ? "open" : ""}`}>
+            <div className="row flex-container">
+              <div className="borderbox">
+                <div>
```

Comment:
why you are using empty div here? you can make span element as a block level element using css.